src/main/clm_initializeMod.F90

```diff
@@ -478,40 +482,66 @@ subroutine initialize2(ni,nj)
           if (finidat_interp_source == ' ') then
              is_cold_start = .true.
              if (masterproc) then
-                write(iulog,*)'Using cold start initial conditions '
+                write(iulog,'(a)')'Using cold start initial conditions '
              end if
           else
              if (masterproc) then
-                write(iulog,*)'Interpolating initial conditions from ',trim(finidat_interp_source),&
-                     ' and creating new initial conditions ', trim(finidat_interp_dest)
+                write(iulog,'(a)')'Interpolating initial conditions from '//trim(finidat_interp_source)
+                write(iulog,'(a)')'Creating new initial conditions file '//trim(finidat_interp_dest)
              end if
           end if
        else
+          if (trim(finidat) == trim(finidat_interp_dest)) then
+             ! Check to see if status file for finidat exists
+             klen = len_trim(finidat_interp_dest) - 3 ! remove the .nc
+             locfn = finidat_interp_dest(1:klen)//'.status'
+             inquire(file=trim(locfn), exist=lexists)
+             if (.not. lexists) then
+                if (masterproc) then
+                   write(iulog,'(a)')' failed to find file '//trim(locfn)
+                   write(iulog,'(a)')' this indicates a problem in creating '//trim(finidat_interp_dest)
+                   write(iulog,'(a)')' remove '//trim(finidat_interp_dest)//' and try again'
+                end if
+                call endrun()
+             end if
+          end if
           if (masterproc) then
-             write(iulog,*)'Reading initial conditions from ',trim(finidat)
+             write(iulog,'(a)')'Reading initial conditions from file '//trim(finidat)
           end if
           call getfil( finidat, fnamer, 0 )
           call restFile_read(bounds_proc, fnamer, glc_behavior, &
                reset_dynbal_baselines_lake_columns = reset_dynbal_baselines_lake_columns)
        end if
     else if ((nsrest == nsrContinue) .or. (nsrest == nsrBranch)) then
        if (masterproc) then
-          write(iulog,*)'Reading restart file ',trim(fnamer)
+          write(iulog,'(a)')'Reading restart file '//trim(fnamer)
        end if
        call restFile_read(bounds_proc, fnamer, glc_behavior, &
             reset_dynbal_baselines_lake_columns = reset_dynbal_baselines_lake_columns)
     end if
 
     ! If appropriate, create interpolated initial conditions
     if (nsrest == nsrStartup .and. finidat_interp_source /= ' ') then
-       is_interpolated_start = .true.
```

Comment:
- [ ] It looks like this variable is no longer used in the code (it looks like I removed its one use at some point but didn't remove the variable). Can you please go ahead and remove it from clm_varctl? I think that, with the changes you have made, its value is no longer correct; rather than fixing it, I feel like we might as well just remove it.